client/src/app/features/messenger/pages/messenger-page/messenger-page.component.html

```
@@ -67,14 +102,18 @@ <h4 class="mt-3 text-center">Nothing here yet, check back later.</h4>
           </div>
         </div>
       </td>
-      <td>{{ message.content }}</td>
+      <td>
+        @if (message.content.length > 35) {
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)